apollo-router/src/plugin/mod.rs

```diff
@@ -104,13 +108,13 @@ impl PluginFactory {
     pub async fn create_instance(
         &self,
         configuration: &serde_json::Value,
-        schema: &str,
+        schema: Arc<String>,
```

Comment:
NIT: Not your doing, but now that we are calling `schema`, `supergraph_sdl` in so many places, maybe we should here as well?

Reply:
Done